lib/src/main/java/com/diffplug/spotless/Formatter.java

```diff
@@ -224,6 +239,11 @@ public String computeLineEndings(String unix, File file) {
 		Objects.requireNonNull(unix, "unix");
 		Objects.requireNonNull(file, "file");
 
+		for (String skipIfContentContain : skipIfContentContains) {
+			if (unix.contains(skipIfContentContain))
+				return unix;
+		}
```

Comment:
The logic might be executed twice across `computeLineEndings` and `compute` but they're both public so to guarantee `skipIfContentContains` are respected, I added it to both.